pkg/proxy/request.go

```diff
@@ -78,6 +78,11 @@ func fromHTTPRequestToListOptions(r *http.Request) (*kubernetes.ListOptions, err
 		opt.Limit = limit
 	}
 
+	// Build phase filter.
+	if phaseVal := params.Get("phase"); phaseVal != "" {
```

Comment:
Do you really need to check if the value is not empty? Default value for string field is empty string, so setting it to the param value w/out checking should not change anything.